openverse_catalog/dags/data_refresh/dag_factory.py

```diff
@@ -0,0 +1,198 @@
+"""
+# Data Refresh DAG Factory
```

Comment:
I'm not sure if there are plans for that in the Catalog either 🤔 At the moment I'm just mirroring the docs for the `provider_dag_factory`, although it could be worthwhile to update them across files. I could try moving this into the doc for the factory method, but it's already quite long.

Reply:
This docstring is fed directly into the DAG's `doc_md` field, gets turned into rendered markdown in the Airflow UI. Here's an example from the TSV loader DAG:

![image](https://user-images.githubusercontent.com/10214785/159340789-5d94e152-5ab7-48cd-aced-af30647e8c78.png)

I think having the docstring as the DAG's documentation is useful, but for DAG factories I've also generated _different_ markdown documentation that includes specifics to the generated DAG (rather than having the same documentation for all DAGs generated by the factory).